docs-ref-services/latest/servicebus-readme.md

```diff
@@ -1,14 +1,13 @@
 ---
 title: Azure Service Bus client library for Python
+description: Azure Service Bus client library for Python
 keywords: Azure, python, SDK, API, azure-servicebus, servicebusservicebus
 author: maggiepint
 ms.author: magpint
 ms.date: 06/08/2021
 ms.topic: article
-ms.prod: azure
-ms.technology: azure
 ms.devlang: python
-ms.service: servicebusservicebus
```

Comment:
This is good to know. Given that this was placed by automation, I'll track this back and get it fixed.